platform/commonUI/general/src/controllers/TimeRangeController.js

```diff
@@ -243,36 +211,12 @@ define(
                 updateTicks();
             }
 
-            function updateStartFromText(value) {
-                try {
-                    updateOuterStart(parseTimestamp(value));
-                    updateBoundsTextForProperty($scope.ngModel, 'end');
-                    $scope.boundsModel.startValid = true;
-                } catch (e) {
-                    $scope.boundsModel.startValid = false;
-                    return;
-                }
-            }
+            function updateFormat(key) {
+                formatter = formatService.getFormat(key) ||
```

Comment:
For the various mct-control definitions, having defaults for when a domain is not provided makes sense; as that's just configuring an input field, and I can see people wanting to control that, or optionally not care about it.

As you say, allowing invalid domains to be used is a correctness concern and should be prevented-- if a user thinks they are entering LST but they're really entering UTC, they would see significantly incorrect data.